mapper/orm-coordination-outbox-polling/src/main/java/org/hibernate/search/mapper/orm/coordination/outboxpolling/cluster/impl/OutboxPollingAgentAdditionalJaxbMappingProducer.java

```diff
@@ -72,23 +74,17 @@ public class OutboxPollingAgentAdditionalJaxbMappingProducer
 			"        </property>\n" +
 			"        <property name=\"totalShardCount\" nullable=\"true\" />\n" +
 			"        <property name=\"assignedShardIndex\" nullable=\"true\" />\n" +
-			// Reserved for future use
-			"        <property name=\"payload\" nullable=\"true\" type=\"materialized_blob\">\n" +
-			// HSEARCH-4727: this column length will be ignored in most dialects, since the blob type is normally unbounded,
-			// but it will force Hibernate ORM to simulate an unbounded BLOB type with DB2.
-			// Using 2147483647 as it's the documented maximum length of BLOBs in DB2:
-			// https://www.ibm.com/docs/en/db2-for-zos/11?topic=types-large-objects-lobs
-			// TODO HSEARCH-4395/HSEARCH-4532 drop this length definition with ORM 6, because ORM 6 will ignore it.
```

Comment:
Did you check this? If true, you might want to adapt `PayloadMappingUtils#MATERIALIZED_BLOB_TEMPLATE`.

Reply:
Double checked it and it has no effect on DB2 (be it present or absent) so I've removed the length. I didn't want to change it initially to keep the "old" value as much unchanged as possible 😃